dist/spdx-tools.cjs

```diff
@@ -679,15 +733,18 @@ class ExternalDocumentReference {
     }
 }
 
+// SPDX-FileCopyrightText: 2023 spdx contributors
+//
+// SPDX-License-Identifier: MIT
 function formatSpdxVersion(spdxVersion) {
-    return "SPDX-" + (spdxVersion ?? "2.3");
+    return `SPDX-${spdxVersion ?? "2.3"}`;
 }
 function generateNamespace(documentName) {
     // Remove/replace invalid characters (https://datatracker.ietf.org/doc/html/rfc2141#section-2.1)
     const formattedDocumentName = documentName
         .replace(/^[^A-Za-z0-9]+/, "")
         .replace(/[^A-Za-z0-9-]/g, "-");
-    return "urn:" + (formattedDocumentName ?? "document") + ":" + uuid.v4();
+    return `urn:${formattedDocumentName || "document"}:${uuid.v4()}`;
```

Comment:
diff contains unrelated changes

Reply:
This is also in one of the built files. But sure, I can move this into a separate diff.